docs/guides/general/remote-access.rst

```diff
@@ -11,303 +11,262 @@ your boot environments reside in encrypted filesystems, SSH access is necessary
+      are designed to simplify providing passwords and snooping console output so that you can interact with unlock processes
+      that are already running in the initramfs. These components are not required for ZFSBootMenu and do not provide a lot of
+      value. If you have no problems installing the module as intended, it is okay to leave the helpers installed. If your
+      distribution has trouble compiling the helpers, just copy the contents of the ``60crypt-ssh`` directory, except for the
+      ``helper`` directory and ``Makefile``, to the modules directory for Dracut. This will most likely be
+      ``/usr/lib/dracut/modules.d/60crypt-ssh``.
 
-If you use Dracut to produce the initramfs images in your boot environment, you may wish to disable the ``crypt-ssh``
-module in those images. Just add
+      If you do not install the contents of ``helper``, you may wish to edit the ``module-setup.sh`` script provided by the
+      package to remove references to installing the helper. At the time of writing, these references consist of the last four
+      lines (five, if you count the harmless comment) of the ``install()`` functioned. Removing these lines should not be
+      critical, as Dracut should happily continue the initramfs creation process even if those installation commands fail.
 
-.. code-block::
+    If you use Dracut to produce the initramfs images in your boot environment, you may wish to disable the ``crypt-ssh``
+    module in those images. Just add::
 
-  omit_dracutmodules+=" crypt-ssh "
+      omit_dracutmodules+=" crypt-ssh "
 
-to a configuration file in ``/etc/dracut.conf.d``. The configuration file must have a ``.conf`` extension to be
-recognized; see `dracut.conf(5) <https://man.voidlinux.org/dracut.conf.5>`_ for more information.
+    to a configuration file in ``/etc/dracut.conf.d``. The configuration file must have a ``.conf`` extension to be
+    recognized; see :manpage:`dracut.conf(5)` for more information.
 
-Configuring Dropbear in ZFSBootMenu
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    To inform ``dracut-network`` that it must bring up a network interface, pass the kernel command-line parameters
+    ``ip=dhcp`` and ``rd.neednet=1`` to your ZFSBootMenu image. If you use another boot loader to start ZFSBootMenu, *e.g.*
+    rEFInd or syslinux, this can be accomplished by configuring that loader. However, it may be more convenient to add these
+    parameters directly to the ZFSBootMenu image::
 
-By default, ``dracut-crypt-ssh`` will generate random host keys for your ZFSBootMenu initramfs. This is undesirable
-because SSH will complain about unknown keys every time you reboot. If you wish, you can configure the module to copy
-your regular host keys into the image. However, there are two problems with this:
+      mkdir -p /etc/cmdline.d
+      echo "ip=dhcp rd.neednet=1" > /etc/cmdline.d/dracut-network.conf
```

Comment:
Won't this add this to both the ZBM image and to the distro/BE image?

Reply:
I think these files need to be explicitly added to images, so they should be OK here.